spec/unit/network/authconfig_spec.rb

```diff
@@ -107,3 +107,33 @@
     end
   end
 end
+
+describe Puppet::Network::AuthConfig do
+  after :all do
```

Comment:
I went with `:all` since each test sets it explicitly as part of the test setup.

Sure, I can change it though.